ballerina-to-openapi/src/main/java/io/ballerina/openapi/service/mapper/OpenAPIResourceMapper.java

```diff
@@ -214,7 +214,7 @@ private Optional<OperationAdaptor> convertResourceToOperation(FunctionDefinition
         Map<String, String> apiDocs = listAPIDocumentations(resource, op);
         //Add path parameters if in path and query parameters
         OpenAPIParameterMapper openAPIParameterMapper = new OpenAPIParameterMapper(resource, op, apiDocs, semanticModel,
-                moduleMemberVisitor, errors, componentMapper);
+                moduleMemberVisitor, errors, componentMapper, openAPI);
         openAPIParameterMapper.getResourceInputs(components, semanticModel);
         if (errors.size() > 1 || (errors.size() == 1 && !errors.get(0).getCode().equals("OAS_CONVERTOR_113"))) {
```

Comment:
Shall we use constant values for OpenAPI diagnostic codes ? Check for other places as well.